client/js/utils/api-utils.js

```diff
@@ -0,0 +1,18 @@
+var ServerActionCreator = require('../actions/server-action-creator');
+var weekData = require('../weekData');
+var _ = require('underscore');
+
+var ApiUtils = {
+  addEvent: function (event) {
```

Comment:
event though it's not the final implementation - if you're making other changes, it'd be cool if you added a callback parameter which would take the ServerActionCreate.eventAdded as the argument. I think it'll set up a better convention for our api-utils. thoughts?